frontend_lib/src/component/ProfileNavigation/ProfileNavigation.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Link } from 'react-router-dom'
+import { PAGE } from '../../helper.js'
+
+const ProfileNavigation = props => {
+  return (
+    <Link
+      to={PAGE.PUBLIC_PROFILE(props.user.userId)}
```

Comment:
I would have expect that this component set some default css for the link, like a mousehover, some font style or a cursor or else.
I don't know if it fit in here

Reply:
I don't think it's a good idea because this component will be used for the avatars too

Reply:
actually, yes, the cursor

Reply:
fixed